exasol_integration_test_docker_environment/lib/docker/images/save/docker_image_save_base_task.py

```diff
@@ -37,9 +37,9 @@ def register_required(self):
     def get_docker_image_task(self):
         raise AbstractMethodException()
 
-    def run_task(self):
-        image_info = self.get_values_from_future(self._image_info_future)
-        tag_for_save = self.get_tag_for_save(image_info)
+    def run_task(self) -> None:
+        image_info: ImageInfo = self.get_values_from_future(self._image_info_future)
```

Comment:
oh and it doesn't complain, because the return type of get_values_from_future should be Any